build/griddle.js

```diff
@@ -494,9 +499,16 @@ return /******/ (function(modules) { // webpackBootstrap
 	        };
 	    },
 	    getFilter: function(){
-	     return ((this.props.showFilter && this.props.useCustomGridComponent === false) ? 
-	        React.createElement(GridFilter, {changeFilter: this.setFilter, placeholderText: this.props.filterPlaceholderText}) : 
-	        "");
+	        if (this.props.showFilter && this.props.useCustomGridComponent === false) {
```

Comment:
I've made this slightly easier to read (at least for me), if you'd like I can revert to the previous formatting.